### PR TITLE
Update BLorient566.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient566.xml [hunk 1]
```diff
@@ -31,74 +31,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            <note>The collection of magic texts is organized in smaller
+                            sub-collections in the manuscript. Some of the texts are accompanied
+                          with the directions for their use.</note>
+                          </msItem>
+                          <msItem xml:id="ms_i1.2">
+                            <locus from="5v"></locus>
+                            <title type="complete" xml:lang="en">Collection of magic prayers</title>
+                            <textLang mainLang="gez"></textLang>
+                            <incipit xml:lang="gez">
+                              <hi rend="rubric">ጸሎተ፡ ግትሐት፡ ሌዊ፡ ሌዊ፡ ሌዊ፡ ሌዊ፡ ሌዊ፡ ሌዊ፡</hi>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient566.xml:62]: ግትሐት in the ms?

File: LondonBritishLibrary/orient/BLorient566.xml [hunk 2]
```diff
@@ -31,74 +31,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            <locus target="#71ra"></locus>
+                            <desc>Small polychrome ornamental band</desc>
+                          </decoNote>
+                        </decoDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="e1">
+                              <locus target="#1 #6 #11 #18 #30 #36 #38 #39 #40 #41 #55 #57 #71"></locus>
+                              <desc type="findingAid">Finding aids, colored pieces of threads in blue
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient566.xml:147]: I think that these are leaf string markers and should be encoded in the description of the binding

File: LondonBritishLibrary/orient/BLorient566.xml [hunk 3]
```diff
@@ -31,74 +31,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            <textLang mainLang="gez"></textLang>
+                            <incipit xml:lang="gez">
+                              <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩ አምላ</hi>ክ፡ ጸሎት፡ በእንተ፡ መፍትሔ፡
+                              ሥራይ፡ ዘተቀድ<hi rend="rubric">ሐ፡ እም፹ወ፩መጻሕፍት፡ ዘወጽአት፡ እምኤሩ</hi>ሳሌም፡ ዘሞዖሙ፡
+                              ለመሠርያን፡ ወነሀብያን፡
+                              ወደጋምያን፡ ለአረሚ፡ ወክርስቲያን፡ ዘፈትሐ፡ ቆጵርያኖስ።
+                            </incipit>
+                            <note>The collection of magic texts is organized in smaller
+                            sub-collections in the manuscript. Some of the texts are accompanied
+                          with the directions for their use.</note>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient566.xml:55]: accompanied by directions, I think

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: LondonBritishLibrary/orient/BLorient566.xml [hunk 4]
```diff
@@ -31,74 +31,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                              <hi rend="rubric">ጸሎተ፡ ግትሐት፡ ሌዊ፡ ሌዊ፡ ሌዊ፡ ሌዊ፡ ሌዊ፡ ሌዊ፡</hi>
+                              ሐጀር፡ ሪማና፡ ፍታሕ፡ ታድሪ፡ ጣዖ፡ ለበሸለሾል፡
+                            </incipit>
+                          </msItem>
+                          <msItem xml:id="ms_i1.3">
+                            <locus from="11rv" to="37va"></locus>
+                            <title type="complete" xml:lang="en">Collection of magic prayers</title>
+                            <textLang mainLang="gez"></textLang>
+                            <incipit xml:lang="gez">
+                              <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አ</hi>ምላክ፡ ጸሎት፡ በእንተ፡ መፍትሔ፡ ሥራይ፡
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient566.xml:71]: I would include መፍትሔ፡ ሥራይ into the title

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**